src/pages/pokedex-site.tsx

```diff
@@ -91,15 +93,16 @@ export default function PokedexSite() {
                     </button>
                 </div>
                 {/* Ensures that there is enough space for the type filter container, so the elements positions below are not affected by it by the state. */}
+                {/* type box  */}
                 <div
-                    className={`col-span-2 mb-8 transition-all duration-500 transform h-[252px] ${
+                    className={`col-span-full desktop:col-span-8 mb-40 desktop:mb-8 transition-all duration-500 transform h-[252px] ${
```

Comment:
Ah I didn't see that before. Comments usually tend to appear in the line right above the line of code that it explains. It would be great if that'd be the case here as well.

Also having both comments does not have any value over having just one of them.